### ISSUE TITLE
RoboDojo

### ISSUE THREAD

**Issue by tuanziqiqi** [OpenGalaxea/GalaxeaVLA#60]
hello, 我注意到G0.5在RoboDojo取得了不错的成绩，请问具体的具体的数据处理，训练设置，推理测评等细节会开源吗？我目前复现的结果并不理想。

**Comment by although-not-but** [comment]
+1 我使用的ce 和 fm loss同时训练 效果不太理想 ，我应该如何调整

**Comment by zhanghaoyu913-cmyk** [comment]
> +1 我使用的ce 和 fm loss同时训练 效果不太理想 ，我应该如何调整
hi ，当前你训练的config是怎么样的

**Comment by although-not-but** [comment]
> > +1 我使用的ce 和 fm loss同时训练 效果不太理想 ，我应该如何调整
> 
> 当前你的config是怎么样的

ce loss = 1 ,fm_loss =1，lr =1e-5，batch=256， step = 30000，flow_samples=4

**Comment by zhanghaoyu913-cmyk** [comment]
> hello, 我注意到G0.5在RoboDojo取得了不错的成绩，请问具体的具体的数据处理，训练设置，推理测评等细节会开源吗？我目前复现的结果并不理想。

hi，对于训练的数据是没有做任何的处理的，之后我们会把这些细节交给robodojo官方开源的

**Comment by tuanziqiqi** [comment]
感谢回复。请问是否会开源RoboDojo的完整训练流程呢？

**Comment by liuuzexiang** [comment]
Hi, could you please clarify whether RoboDojo post-training uses checkpoints/g05-base/checkpoints/model_state_dict.pt as the pretrained checkpoint?

I’m currently trying to reproduce the results, but the performance gap is quite large. The success rate I’m getting is only 7.7%, which is significantly lower than the reported 14.8%. 

I also noticed that the official RoboDojo code includes a G05 post-training config. After taking a look, I’m a bit confused about the action_state_transforms setting:

In the pretrain config, action_state_transforms is set to RelativeJointTransform
In the post-train config, it is set to null

Could you confirm whether the post-training checkpoint is indeed initialized from the publicly released pretrain checkpoint? Also, is it expected that action_state_transforms changes from RelativeJointTransform during pretraining to null during post-training?

I also tried setting it to null for post-training as in the official config, but the performance became even worse. Could you help clarify the correct configuration or whether there are any other differences between the released pretrain checkpoint and the checkpoint used for the reported post-training results?
